webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -43,6 +44,28 @@ let GitBlameInfoModal = React.createClass({
         );
     },
 
+    /**
+     * @returns {*} The row of label:link to display in the modal
+     */
+    displayScreenshotLink() {
+        const label = this.props.intl.formatMessage({ id: "textUnit.gitBlameModal.screenshots"});
+        const branchScreenshots = this.getBranchScreenshots();
```

Comment:
you can make `getBranchScreenshots()` return non `null` result so that you don't have to test it